.github/scripts/get-version.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash -e
 
-grep -Po "val stableVersion = \"\K[0-9]+.[0-9]+.[0-9]+" version.gradle.kts
+grep "val stableVersion = " version.gradle.kts | grep -Eo "[0-9]+.[0-9]+.[0-9]+"
```

Comment:
I tested both on debian and macos...the `grep -p` one fails on a mac, so that seems like a worthy improvement when developing locally....

Reply:
for some reason `-P` used to work for me (windows / git bash) but no longer 🤷‍♂️

Reply:
`-P` is probably a GNU grep extension. Try running `grep --version`. Mac uses `grep (BSD grep, GNU compatible) 2.6.0-FreeBSD` `brew install grep` installs gnu grep as `ggrep`.